### PR TITLE
Added topic for Visual Basic null-conditional operators 

### LINKED ISSUE #7559: Visual Basic Guide should reference null-conditional operators article
The one for both (C# and Visual Basic) [article](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-conditional-operators) about null-conditional operators `?.` and `?[]` is referenced only once from the Visual Basic Guide, in the [Visual Basic 14](https://docs.microsoft.com/en-us/dotnet/visual-basic/getting-started/whats-new#visual-basic-14) section of the "What's new for Visual Basic" article.

I think the operator page also should be referenced from [Operators](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/operators/) or one of its sub-articles. Also, related TOC should be updated:
https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/operators/operators-listed-by-functionality

/cc @rpetrusha

### PR DESCRIPTION
**Pull request by rpetrusha**
## Added topic for Visual Basic null-conditional operators 

This PR:
- Replaces the combined C#/VB null-conditional operators topic with separate topics.
- Modifies the link in the Visual Basic what's new topic to point to the VB topic.
- Updates index- and TOC-related content to include the null-conditional topic.

Fixes #7559 

//cc @svick @KathleenDollard

### CODE REVIEW

**Review by BillWagner — APPROVED** [review]
This LGTM @rpetrusha 

I left one comment, which may reflect my limited VB knowledge. I don't know if any changes are needed or not, but once you consider, you can :shipit:

File: docs/visual-basic/language-reference/operators/null-conditional-operators.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  - "?[] operator [Visual Basic]"
+---
+# ?. and ?() null-conditional operators (Visual Basic)
+
+Tests the value of the left-hand operand for null (`Nothing`) before performing a member access (`?.`) or index (`?()`) operation; returns `Nothing` if the left-hand operand evaluates to `Nothing`.
+
+These operators help you write less code to handle null checks, especially when descending into data structures. For example:
+
+```vb
+Dim length = customers?.Length  ' Nothing if customers is Nothing
```
[the remark this answers is not in the document]
Comment (KathleenDollard) [reply docs/visual-basic/language-reference/operators/null-conditional-operators.md]: Agree with Bill. Let's put the type here like we do in the C# topic

**Review by KathleenDollard — COMMENTED** [review]
Looks good

File: docs/visual-basic/language-reference/operators/null-conditional-operators.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rpetrusha) [reply docs/visual-basic/language-reference/operators/null-conditional-operators.md]: Thanks -- it's not obvious. I've revised it slightly and added the type.

**Review by KathleenDollard — APPROVED** [review]
Excellent!

**Pull request merged**